recipes/uxarray/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: uxarray
+  version: {{ version }}
+
+source:
+  url: https://github.com/UXARRAY/uxarray/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: 3cb04f27168ac946ae189fef55f95e19a4b6248c9af847d8ed5c699d3c24b6c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
```

This looks like it's pure Python, so could use noarch instead and drastically cut down the number of builds:
```suggestion
  noarch: python
```